core/src/connection/pool.rs

```diff
@@ -1058,6 +1062,16 @@ impl ConnectionLimits {
         self
     }
 
+    /// Configures the maximum number of concurrent established connections (both inbound and
+    /// outbound).
+    /// Note: This should be used in conjunction with
+    /// [`ConnectionLimits::with_max_established_incoming`] to prevent possible eclipse attacks (all connections being
+    /// inbound.
```

Comment:
```suggestion
    /// Configures the maximum number of concurrent established connections (both
    /// inbound and outbound).
    ///
    /// Note: This should be used in conjunction with
    /// [`ConnectionLimits::with_max_established_incoming`] to prevent possible
    /// eclipse attacks (all connections being inbound).
```